authentication.py

```diff
@@ -0,0 +1,92 @@
+import re
+from passlib.context import CryptContext
+from fastapi import HTTPException, status
+import jwt
+
+from models import User
+from config import get_settings
+
+pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
+
+
+def get_hashed_password(password):
+    return pwd_context.hash(password)
+
+
+async def very_token(token: str):
+    '''verify token from login'''
+    try:
+        payload = jwt.decode(token, get_settings().SECRET,
+                             algorithms=["HS256"])
+        user = await User.get(id=payload.get("id"))
+
+    except:
```

Comment:
**Static Code Issue**: Do not use bare 'except' <br> **Fix**: Specify the exception type. <br> **Code Suggestion**: 
 ```
 - except:
 + except jwt.ExpiredSignatureError:
 ```